### DISCUSSION TITLE
Installing mayaUSD plugin in a studio

### DISCUSSION THREAD

**Discussion by Lucas3Dspain** [Autodesk/maya-usd#2114] [Q&A] [answered]
**First of all this is more like a question than a feature request, I didn't know where to post it.**
My Issue is that we are trying to use the mayaUSD plugin in a studio production environment. So we had to install the plugin manually on every workstation and now I'm wondering what is gonna happen when we want to update to the next version. Will we need to reInstall manually on every computer again? we don't like that :( 

Is there a way to have mayaUSD plugin files in our server and point them with some environment variables or something similar?

We will be cool if we had to uninstall the version we had installed on every workstation, and then use the environment variables to point to the server plugin files.

Thanks

**Comment by seando-adsk** [comment, marked as answer]
Yes that is something supported by Maya plugins.

This help page: https://help.autodesk.com/view/MAYAUL/2022/ENU/?guid=Bifrost_MayaPlugin_install_bifrost_for_maya_load_from_custom_location_html
gives directions on how to do that for the Bifrost plug-in, but it is the same for the MayaUsd plugin.

Basically it involves copying the MayaUsd folder from one of the machines it is installed on to a network location. Then on each machine set the env var MAYA_MODULE_PATH to point to that folder. Then you can uninstall MayaUsd from those machines.

Sean
